app/javascript/availability/index.js

```diff
@@ -484,8 +491,7 @@ const availability = {
     return (
       ['UP-MICRO'].includes(holding.libraryID) &&
       holding.homeLocationID !== 'THESIS-NML' &&
-      holding.itemTypeID === 'MICROFORM' &&
-      !(holding.locationID in availability.illiadLocations)
+      holding.itemTypeID === 'MICROFORM'
```

Comment:
I think the change here is causing the test to fail.  Do we want to handle all MICROFORM links the same now, and not differentiate between ILL and non-ILL locations?  If so we'll have to remove some tests: https://github.com/psu-libraries/psulib_blacklight/blob/main/spec/javascript/availability/components/special_request_link.jsx.spec.js#L221

Reply:
I am assuming we will eventually not link to ILL from the locations since this no recall change is updating I want It button to go to myaccount's ILL page. But I will put this line back for the initial pass of this PR and have Ruth review things and give us feedback re linking the ILL locations or not.